README.md

```diff
@@ -25,24 +25,24 @@ Rolling | Garden (not released) | [ros2](https://github.com/gazebosim/ros_gz/tre
 This repository holds packages that provide integration between
 [ROS](http://www.ros.org/) and [Gazebo](https://gazebosim.org):
 
-* [ros_ign](https://github.com/gazebosim/ros_gz/tree/ros2/ros_ign):
+* [ros_gz](https://github.com/gazebosim/ros_gz/tree/ros2/ros_gz):
```

Comment:
Just leaving a note that these links will be broken until the packages are migrated